rolling/distribution.yaml

```diff
@@ -6168,7 +6168,7 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/rmw_desert-release.git
-      version: 1.0.1-1
+      version: null
```

Comment:
The rosdistro formatter doesn't like explicit `null` and prefers omission. We can have a separate conversation about allowing explicit nulls for cases like this, but for now I think we should remove the line to unblock builds.

```suggestion
```